apps/testnorge-arena/arena-core/src/main/java/no/nav/registre/arena/core/consumer/rs/command/PostEndreInnsatsbehovCommand.java

```diff
@@ -0,0 +1,54 @@
+package no.nav.registre.arena.core.consumer.rs.command;
+
+import java.util.concurrent.Callable;
+
+import lombok.extern.slf4j.Slf4j;
+import no.nav.registre.arena.core.consumer.rs.request.EndreInnsatsbehovRequest;
+import no.nav.registre.arena.core.consumer.rs.response.EndreInnsatsbehovResponse;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+
+import static no.nav.registre.arena.core.consumer.rs.util.Headers.CALL_ID;
+import static no.nav.registre.arena.core.consumer.rs.util.Headers.CONSUMER_ID;
+import static no.nav.registre.arena.core.consumer.rs.util.Headers.NAV_CALL_ID;
+import static no.nav.registre.arena.core.consumer.rs.util.Headers.NAV_CONSUMER_ID;
+
+@Slf4j
+public class PostEndreInnsatsbehovCommand implements Callable<EndreInnsatsbehovResponse> {
+
+    private final WebClient webClient;
+    private final EndreInnsatsbehovRequest request;
+
+    private static final String endrePath = "/v1/endreInnsatsbehov";
+
+    public PostEndreInnsatsbehovCommand(EndreInnsatsbehovRequest request, WebClient webClient) {
+        this.webClient = webClient;
+        this.request = request;
+    }
+
+    @Override
+    public EndreInnsatsbehovResponse call() {
+        EndreInnsatsbehovResponse response = null;
+        try {
+            response = webClient.post()
+                    .uri(builder ->
```

Comment:
Du trenger ikke builder `.uri(endrePath)`

Reply:
Jo, ganske sikker på at man trenger builder siden det bygger på `baseUrl` som blir satt for webclient i `InnsatsArenaForvalterConsumer`